src/core/background/util.ts

```diff
@@ -61,6 +65,32 @@ export async function filterInactiveTabs(activeTabs: ManagerTab[]) {
 	});
 }
 
+/**
+ * @param tabId - tab to get song from
+ * @returns the details about the channel of the currently playing song in tab
+ */
+export async function getChannelDetails(tabId: number) {
+	return sendBackgroundMessage(tabId, {
+		type: 'getChannelDetails',
+		payload: undefined,
+	});
+}
+
+/**
+ * Is current channel blocklisted
+ *
+ * @param channelID - ID of the channel to check
+ * @param connector - Details about the connector to check
+ * @returns true if current channel is blocklisted; false otherwise
+ */
+export async function isChannelBlocklisted(
+	channelID: string,
```

Comment:
I guess we should also rename this one to `channelId` too - and then we can merge :tada: